llama.cpp

```diff
@@ -7908,6 +7909,7 @@ struct llm_tokenizer_spm {
         // split string into utf8 chars
         int index = 0;
         size_t offs = 0;
+        symbols.reserve(text.size());
```

Comment:
We should also clear symbols here to be correct - currently we construct a tokenizer and then immediately tokenize, so it doesn't make a functional difference.